lib/Service/ReplicateAPIService.php

```diff
@@ -68,10 +67,9 @@ public function createWhisperPrediction(string $audioFileContent, bool $translat
 	 * @param File $file
 	 * @param bool $translate
 	 * @return string
-	 * @throws AppConfigTypeConflictException
+	 * @throws GenericFileException
```

Comment:
Does getValueString not throw here?

Reply:
The only appConfig exception that could bubble up to there is in the request method that catches everything.

Reply:
`appConfig->getValueString` is called in `createImagePrediction`.